table.csv

```diff
@@ -111,6 +111,7 @@ ws,                             multiaddr,      0x01dd,         permanent,
 wss,                            multiaddr,      0x01de,         permanent,
 p2p-websocket-star,             multiaddr,      0x01df,         permanent,
 http,                           multiaddr,      0x01e0,         draft,
+kcp,                            multiaddr,      0x01e1,         draft, Reliable, Ordered UDP with CFB encryption, Reed Solomon Forward Error Correction, and Connection Multiplexing and automatic tuning
```

Comment:
```suggestion
kcp,                            multiaddr,      0x01e1,         draft, Reliable; ordered; encrypted and error-checked UDP with multiplexing and auto-tuning
```

the commas are going to be causing a problem since this is a simplistic CSV table (i.e. quotes won't help)